### PR TITLE
add antlr visitor for MySQL DCLStatement

### PR DESCRIPTION
**Pull request by tianbin1001**
add antlr visitor for MySQL DCLStatement

### CODE REVIEW

File: sharding-sql-test/src/main/resources/sql/visitor/dcl/rename_user.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<sql-cases>
+    <sql-case id="rename_user" value="RENAME USER 'user_dev'@'localhost' TO 'user_dev'@'127.0.0.1'" db-types="MySQL" />
+    <sql-case id="rename_users" value="RENAME USER user1_bak TO user1, user2_bak TO user2" db-types="MySQL" />
+
```
Comment (tristaZero) [inline sharding-sql-test/src/main/resources/sql/visitor/dcl/rename_user.xml]: Please delete blank line.

Comment (tianbin1001) [reply sharding-sql-test/src/main/resources/sql/visitor/dcl/rename_user.xml]: done

**Pull request merged**